### DISCUSSION TITLE
RecognizeDestinationPrefixes

### DISCUSSION THREAD

**Discussion by JezhikLaas** [LuckyPennySoftware/AutoMapper#4127] [NA]
### Source/destination types

```csharp
public class Source { public int Member { get; set; } }

public class Target { public int M_Member { get; set; } }
```

### Mapping configuration

```csharp
private static readonly MapperConfiguration Configuration = new(configuration =>
{
    configuration.CreateMap<Source, Target>().ForAllMembers(_ => configuration.RecognizeDestinationPrefixes("M_"));
});
```

### Expected behavior

The following test used to work before 12.0.0. It fails now and I do not see anything related in the What's new.

### Actual behavior

Mapping does not happen anymore

### Steps to reproduce

```csharp
[Fact]
public void ShouldUseDestinationPrefix()
{
    var mapper = Configuration.CreateMapper();
    var source = new Source { Member = 666 };
    var target = mapper.Map<Target>(source);

    target.M_Member.Should().Be(source.Member);
}
```

**Comment by lbargaoanu** [comment]
I'm afraid that this code doesn`t make much sense. Check our tests for proper usage. If it still doesn't work, I'll take a look.

**Reply by JezhikLaas** [reply]
I have checked your tests and realized that my usage of ```RecognizeDestinationPrefixes``` was wrong. Seems to have worked just by accident in the older releases. 
Thanks, I have it working now.
